### PR TITLE
Add Redis to the application

### PR DESCRIPTION
**Pull request by chromevoid**
```
Now Redis is working locally.

Code coverage:
Name        Stmts   Miss  Cover
-------------------------------
models.py     120     25    79%
server.py     125     15    88%
-------------------------------
TOTAL         245     40    84%
```

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -197,43 +197,71 @@ def delete_inventories(inventory_id):
...
 
 ######################################################################
 # QUERY INVENTORIES
 ######################################################################
-@app.route('/inventories/query', methods = ['GET'])
+@app.route('/inventories/query', methods=['GET'])
 def query_inventories_by_name_status():
-    name = request.args.get('name').strip()
-    status = request.args.get('status').strip()
-
```
Comment (monabest) [inline server.py]: strip() is wrong? I was thinking to remove redundant spaces.

Comment (chromevoid) [reply server.py]: No, strip() is not wrong. I just think that strip() is not necessary for now, so I removed it.

Comment (chromevoid) [reply server.py]: The problem is that I need to serialize the inventory before using the [x for x in A if x in B] statement.

```
results1 = [inventory.serialize() for inventory in inventories_by_name]
results2 = [inventory.serialize() for inventory in inventories_by_status]
results = [r for r in results1 if r in results2]
```

File: server.py [hunk 2]
```diff
@@ -197,43 +197,71 @@ def delete_inventories(inventory_id):
...
+    status = request.args.get('status')
+
     # query by name and status
-    inventories_by_id = Inventory.find_by_name(name)
+    inventories_by_name = Inventory.find_by_name(name)
     inventories_by_status = Inventory.find_by_status(status)
-    if not inventories_by_status or not inventories_by_id :
+    if not inventories_by_status or not inventories_by_name:
         raise NotFound("Query Inventory with name '{}' and status '{}'  was not found.".format(name, status))
-    results = [inventory.serialize() for inventory in inventories_by_id  if
```
Comment (chromevoid) [inline server.py]: Here's the location of the problem that caused the failure of the test case.

**Pull request merged**